_posts/blog/2018-11-12-Latin@s-in-Ohio-review

```diff
@@ -0,0 +1,76 @@
+but as a fellow Midwestern Latina, I felt a strong connection to everyone’s stories. In the late 90s, my family moved from the 
+east coast to a city that is less than a few miles from Ohio. Many of the places mentioned in the videos, I recognized due to 
+my family’s frequent visits to these locations. To say the least, I was thrilled to engage with this material. 
+
+Coming from an multi-ethnic and biracial family, and moving from New York to the Midwest was a challenge for my family. We 
+left everything we were accustomed to, for a place where we did not see or hear people like us, nor did most people understand 
+our backgrounds. Therefore, as I watched the videos, I found myself agreeing with several of the comments made of 
+participants’ lived experiences and values, because to me it represented my families and our reality in the Midwest at a 
+specific moment in time. As a result, I felt connected to all of the individuals and proud to hear their stories. 
+           
+In addition, I was excited to find the majority of narratives were in Spanish, which was surprising due to the fact that 
+English is the dominant language in the Midwest. This important decision represented Dr. Foulis’ centering and celebration of 
+the Spanish language and Latinx culture, which was emphasized by the lack of English translations for the videos and shown by 
+the heritage speakers proudly using the language they are most comfortable with. This collection of video narrative is an 
+authentic example how the archive can bring to the forefront the voices of those usually excluded and marginalized to support 
+their efforts to preserve their stories.  
+
+Returning to my reactions as I read, I couldn’t help, but question my excitement. Why was I drawn to these stories, even 
+though I frequently and intentionally visited and engaged with Latinxs spaces? After reading “ To Suddenly Discover Yourself 
+Existing” by Caswell, et al., the aha moment appeared and they provided the words to describe my emotions. The authors 
+introduced symbolic annihilation and representational belonging and their meanings resonated with me. They defined symbolic 
+annihilation as the following “to be an eternal outsider whose very existence is presumed an impossibility”(Caswell et al. 
+58), and representational belonging “describes the affective responses community members have to seeing their communities 
+presented with complexity and nuance” (Caswell et al. 75). This resonated with me because I never realized that the stories of 
+my small, local marginalized community could be collected, seen as valued, and recorded for others to learn from our lived 
+experiences and for us to sustain our culture. This collection of video narratives Latinxs matter and we contribute valuable 
+perspective and knowledge to what it means to be Latinx in Ohio. 
+
+Nevertheless, I am left pondering what are the implications of this collection being held and supported by an institution of 
```

Comment:
Grammar: remove "what are" (so it just says "I am left pondering the implications...") or turn this into a question.